azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -124,6 +127,10 @@ public ExecutorManager(final Props azkProps, final ExecutorLoader loader,
     this.loadRunningFlows();
 
     this.queuedFlows = new QueuedExecutions(azkProps.getLong(AZKABAN_WEBSERVER_QUEUE_SIZE, 100000));
+
+    // The default threshold is set to 30 for now, in case some users are affected. We may
+    // decrease this number in future, to better prevent DDos attacks.
+    this.maxConcurrentRunsOneFlow = azkProps.getInt(AZKABAN_MAX_NUM_CONCURRENT_FLOW, 30);
```

Comment:
make 30 a variable

Reply:
if a negative number or 0 is set, should we throw exception?

Reply:
Added a static variable to place the default number.
Checking number validity. we may bring up a util function to cover all similar cases. Should refactor in future.